multiversx_sdk_cli/config.py

```diff
@@ -249,6 +249,8 @@ def determine_final_args(argv: List[str], config_args: Dict[str, Any]) -> List[s
 def get_dependency_directory(key: str, tag: str) -> Path:
     parent_directory = get_dependency_parent_directory(key)
     if tag == 'latest':
+        if not parent_directory.is_dir():
+            return parent_directory / tag
```

Comment:
Is this a change related to pipx etc.?

Reply:
Not really. Randomly tested the `mxpy deps check testwallets` command and there was an error when the dependency was not installed. Since the tag is `latest` it used to do `Path("~/multiversx-sdk/testwallets").iterdir()` and since the folder did not exist it threw an error on `os.listdir()` in `pathlib`.